pages/index.js

```diff
@@ -20,7 +20,7 @@ export default function Index({ posts, globalData }) {
 
   return (
     <Layout>
-      <div className={styles.wrapper}>
+      <div className={`${styles.wrapper} before:bg-white before:bg-white before:backdrop-blur-lg before:dark:bg-black before:dark:bg-opacity-30 before:bg-opacity-10 `}>
         <DynamicReactPlayer
```

Comment:
We should be only adding the post, not changing styles so we don't interfere with the other ticket

Reply:
```suggestion
      <div className={styles.wrapper}>
```